net/DevExtreme.AspNet.Data/Aggregation/SumAggregator.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DevExtreme.AspNet.Data.Aggregation {
+
+    class SumAggregator : Aggregator {
+        decimal? _sum;
+
+        public override void Step(object value) {
+            if(value != null) {
+                if(!_sum.HasValue)
+                    _sum = 0;
+
+                try {
+                    _sum += Convert.ToDecimal(value, CultureInfo.InvariantCulture);
+                } catch(FormatException) {
+                } catch(InvalidCastException) {
```

Comment:
Use matching `catch(Exception ex) when (ex is FormatException || ex is InvalidCastException) { }` instead.